examples/pytorch/conda.yaml

```diff
@@ -9,6 +9,6 @@ dependencies:
   - pip
   - pip:
     - torch==1.6.0
-    - torchvision==0.5.0
+    - torchvision==0.7.0
```

Comment:
`torchvision==0.7.0` should be compatible with `torch==1.6.0` according to the table in https://github.com/pytorch/vision#installation